client/app/account/hooks/useChangePassword.ts

```diff
@@ -0,0 +1,53 @@
+import { useRouter } from 'next/navigation'
+import { useForm } from 'react-hook-form'
+import { useToast } from '@/_hooks'
+import { isErrorWithMessage } from '@/_utils'
+import { useChangePasswordMutation } from './api'
+import { PasswordSchema, passwordResolver } from '../schema'
+
+export const useChangePassword = () => {
+  const router = useRouter()
+  const { toast } = useToast()
+
+  const form = useForm<PasswordSchema>({
+    resolver: passwordResolver,
+    defaultValues: {
+      currentPassword: '',
+      newPassword: '',
+      confirmNewPassword: '',
+    },
+  })
+
+  const { mutate, isPending } = useChangePasswordMutation()
+
+  const onChansePassword = (data: PasswordSchema) => {
+    mutate(data, {
```

Comment:
You don't need to pass `confirmNewPassword`. You can just pass except it!

Reply:
Got it! Thanks!
d2a454df2daf2d0a1857e64502c18f254a7a3d6d